utils/docker-utils.sh

```diff
@@ -273,30 +304,7 @@ docker::deploy_sanity() {
     fi
 
     for i in "$@"; do
-        log info "Waiting for $i to run ..."
-        local start_time
-        start_time=$(date +%s)
-
-        error_message=()
-        until [[ $(docker::get_current_service_status "${i}") == *"Running"* ]]; do
-            config::timeout_check "${start_time}" "$i to run"
-            sleep 1
-
-            new_error_message=($(docker::get_service_unique_errors "$i"))
-            if [[ -n ${new_error_message[*]} ]]; then
-                # To prevent logging the same error
-                if [[ "${error_message[*]}" != "${new_error_message[*]}" ]]; then
-                    error_message=(${new_error_message[*]})
-                    log error "Deploy error in service $i: ${error_message[*]}"
-                fi
-                # To exit in case the error is not having the image
-                if [[ "${new_error_message[*]}" == *"No such image"* ]]; then
-                    log error "Do you have access to pull the image?"
-                    exit 124
-                fi
-            fi
-        done
-        overwrite "Waiting for $i to run ... Done"
+        docker::await_container_status "$i" "Running"
```

Comment:
```suggestion
        docker::await_service_status "$service_name" "Running"
```

Reply:
I will consider changing the name of this function in another PR, because it will affect 7 files